pages/api/graphql-rest.page.ts

```diff
@@ -103,8 +118,8 @@ const resolvers: Resolvers = {
         mailing,
         format,
         filter,
-        labelType,
-        sort,
+        labelType ?? undefined,
+        sort ?? undefined,
```

Comment:
I had to add this here as I was getting a TS error about it being ```null```, even though I'm passing ```undefined```...

Reply:
Here's what I get when I remove the ```?? undefined```
<img width="558" alt="Screen Shot 2021-04-21 at 4 43 52 PM" src="https://user-images.githubusercontent.com/39680460/115618529-d374cd80-a2c0-11eb-82a8-5db21c2c23a0.png">
Does the ```Maybe``` mean it could be ```null``` as well?

Reply:
Ya codegen made `Maybe` and it means it can be null.

Reply:
So would it be better to leave it how it is, or remove the ```?? undefined``` and just add ``` | null``` to the type below?

Reply:
GraphQL seems to operate having `null` as the only nullish value (it doesn't have undefined like JS). Maybe it would be best to use `labelType: string | null,` on the server side instead of `labelType?: string,` (even though that's more concise) so you can just keep the `null` instead of having to convert to/from `undefined`